### DISCUSSION TITLE
creating another chatgpt response column

### DISCUSSION THREAD

**Discussion by valatas37** [AntlerVC/firetable#1273] [Support / Q&A] [answered]
If I would like to create another chatgpt response column, what should I do. I tried to copy the column configuration of the existing chatgpt response alter the listener field and change the relevant field name in the derivative script, but it did not work.

**Comment by harinij** [comment, marked as answer]
@valatas37 you can create a column of type `derivative` and then ensure the build of it is successful on save. 

Here are some pointers:
[How to use derivatives](https://docs.rowy.io/field-types/derivative)
[How to check the cloud logs](https://docs.rowy.io/cloud-logs#accessing-cloud-logs)

Let us know if you are still facing the issue and if you can share the script we can double check as well.

**Reply by valatas37** [reply]
Hi @harinij . It works. YOu are the best. Would it be possible to use gpt-4. Do I need to join waitlist API first if I would like to use GPT-4 within parallelgpt?

**Reply by BeeBombshell** [reply]
Hi, @valatas37! Yes, it is possible to use GPT-4 within ParallelGPT. However, you do need to join the waitlist for the OpenAI API first. Once you have joined the waitlist and been approved, you can then modify the derivative script to use GPT-4 API.
Here's the link to join the GPT-4 waitlist: https://openai.com/waitlist/gpt-4-api
